### DISCUSSION TITLE
Prevent login on registration?

### DISCUSSION THREAD

**Discussion by dambridge** [jacksleight/statamic-members#21] [Q&A]
Just starting with implementing this. I see that when a new user registers, they're automatically logged in. This would provide them with immediate access to protected content. Is it possible to *not* authenticate first and (thinking out loud here) notify them with the email linking to the activation form?

**Comment by jacksleight** [comment]
Hey. The registration process is actually all handled by Statamic itself, and there isn't currently any way to alter the behaviour. However that would be a useful feature to have. I'd recommend submitting it to the ideas repo (https://github.com/statamic/ideas).

**Reply by dambridge** [reply]
Cheers!

**Reply by dambridge** [reply]
What I did actually try was creating a `pending_members` group as the default for registration, and just look for a `members` group for access in the front end instead of `logged_in`. The thought being that the admin could then just add the user to the members group after review and approval in the CP.

But unfortunately, simply adding users to the `members` group would not allow for the Memberbox functionality in the CP to send them an invite to activation. I'll follow up on the Statamic site. Thanks!
